easybuild/easyconfigs/c/cdat/dv3d-pkg_resource-paths.patch

```diff
@@ -0,0 +1,60 @@
+diff -ru dv3d-8.2.1/DV3D/ButtonBarWidget.py dv3d-8.2.1-edit/DV3D/ButtonBarWidget.py
```

Comment:
Please add patch description + link to upstream source or mention patch author (+ fix SHA256 for this patch file)